Tools/autotest/locations.txt

```diff
@@ -52,3 +52,5 @@ UChicago=41.7886079,-87.59871329999999,191,0
 Yachiyo=35.770481,140.107929,36,90
 Tyndall=30.0142232,-85.4984701,7,0
 Elvenes=68.871422,17.986690,17,256
+Kawachi=35.879129,140.339683,7,0
```

Comment:
The nose is not pointing towards the riverbed field.
I think it would be better to point the nose towards the field.